api/XDS_PROTOCOL.md

```diff
@@ -345,6 +345,11 @@ resources to avoid resending them over the network.
 
 ![Incremental reconnect example](diagrams/incremental-reconnect.svg)
 
+#### Resource names
+Resources are identified by a resource name or an alias. Aliases of a resource, if present, can be identified by the alias field in the resource of a DeltaDiscoveryResponse. The resource names
+will be returned in either the name field in the resource of a DeltaDiscoveryResponse or in the actual body
```

Comment:
I'm a bit confused by this second sentence; are we not going to be prescriptive about how to use the new `name` field? Ideally we move away from duck typing the name field inside the body and use the new name/aliases.

Reply:
Agreed. Was more thinking of encompassing current functionality with ideal, but let me remove that. This is incremental specific.

Reply:
Done. Thanks.